usaspending_api/api_contracts/contracts/v2/references/data_dictionary.md

```diff
@@ -0,0 +1,38 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Data Dictionary [/api/v2/references/data_dictionary/]
+
+This endpoint powers USAspending.gov's data dictionary page.
+
+## GET
+
+This endpoint returns data corresponding to the latest data dictionary csv file.
+
++ Response 200 (application/json)
+    + Attributes
+        + `document` (DataDictionary)
```

Comment:
Typically, there isn't a benefit defining a data structure as the entire response. It is beneficial when the response is an array of objects, or could be reused.

In this case, the entire DataDictionary object could be listed as the Response. This doesn't need to change for this PR, but it shouldn't become the standard.